benchmark/Dockerfile

```diff
@@ -5,10 +5,11 @@ RUN apk add --no-cache make git bash wget
 RUN apk add --no-cache gcc musl-dev python3-dev libffi-dev openssl-dev
 RUN apk add --update --no-cache py3-numpy py3-scipy py3-pillow
 RUN apk add --update --no-cache gfortran freetype-dev libpng-dev openblas-dev g++ py3-numpy-dev
-RUN apk add --no-cache go
+RUN apk add --no-cache go rust cargo
 
 RUN pip install --upgrade pip pipenv
 RUN pip install aea[all] --upgrade --force-reinstall
 
 RUN wget https://raw.githubusercontent.com/fetchai/agents-aea/main/Pipfile
 RUN pipenv install -d --deploy --skip-lock --system
+RUN pip install aea-ledger-fetchai
```

Comment:
see below, this might be not the right thing to do

Reply:
But the question is should we run the benchmark against pypi version or not. I think we should.

Reply:
everythin can be updated inside the container later so not a problem, at least i want to be sure dependencies are installed ok